JS-Algorithms/challenges.js

```diff
@@ -13,7 +13,17 @@ Invoking "readableTime(3690)" should return "01:01:30" (HH:MM:SS)
 ***** */
 
 const readableTime = (seconds) => {
-  // YOUR CODE HERE...
+  // get hours, minutes, seconds
+  let hours = Math.trunc(seconds / 60 / 60) 
+  let minutes = Math.trunc((seconds / 60) - (hours * 60))
+  let secondsLeft = seconds - (hours * 60 * 60) - (minutes * 60) 
+  
+  // display values as 2 digits
+  let dHours = hours.toLocaleString('en-US', {minimumIntegerDigits: 2, useGrouping:false})
```

Comment:
found about this here on the second answer. [https://stackoverflow.com/questions/8043026/how-to-format-numbers-by-prepending-0-to-single-digit-numbers](https://stackoverflow.com/questions/8043026/how-to-format-numbers-by-prepending-0-to-single-digit-numbers)

however regarding current user settings we are hardcoding en-US region. I see on the docs what you say about it automatically using the user settings. "In implementations without Intl.DateTimeFormat support, this parameter is ignored and the host's locale is usually used." So this would actually be a an issue if the env doesn't have this. Although very unlikely. 

[https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/toLocaleString](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/toLocaleString)